File: iotcloud-restapi-java/Core/src/com/qcloud/restapi/iotcore/response/CreateMultiDeviceResponse.java
package com.qcloud.restapi.iotcore.response;

import com.google.gson.*;

public class CreateMultiDeviceResponse extends Response {
    /**
     * 解析JSON字串并构造CreateMultiDeviceResponse对象
     *
     * @param response 从服务端返回的应答JSON字串
     */
    public CreateMultiDeviceResponse(String response) {
        super(response);
    }

    public String getTaskID() {
        try {
            if (jsonObj != null) {
                return jsonObj.get("taskID").getAsString();
            }
        }catch (Exception e) {
        }

        return "";
    }
}

File: iotcloud-restapi-php/Core/src/IoTCloud/Response/CreateMultiDeviceResponse.php
<?php

namespace TXIoTCloud\Response;

require_once "BaseResponse.php";

class CreateMultiDeviceResponse extends BaseResponse
{

    /**
     * 任务ID，腾讯云生成全局唯一的任务 ID
     */
    protected $taskID;

    /**
     * CreateMultiDeviceResponse constructor.
     */
    public function __construct()
    {
    }

    /**
     * 获取任务ID
     */
    public function getTaskID()
    {
        return $this->taskID;
    }

    /**
     * 设置任务ID
     * @param $taskID
     */
    public function setTaskID($taskID)
    {
        $this->taskID = $taskID;
    }


}